ui/shared/components/panel/variants/Frequencies.jsx

```diff
@@ -40,9 +40,13 @@ FreqLink.propTypes = {
 }
 
 const getFreqLinkPath = ({ chrom, pos, variant, value }) => {
-  const isRegion = parseFloat(value, 10) <= 0
+  const floatValue = parseFloat(value, 10)
+  const isRegion = floatValue <= 0
   let coords
-  if (isRegion) {
+  // eslint-disable-next-line no-restricted-globals
```

Comment:
From this article (https://stackoverflow.com/questions/46677774/eslint-unexpected-use-of-isnan) it looks like if we use `Number.isNaN` this will work without throwing a linter warning. Suppressing linter warnings is totally okay to do, but if there is an easy workaround like this one its better to use it. But if that doesn't actually work than you should leave this as is